client/components/Info Panel/InfoPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { RiFolderUploadFill } from 'react-icons/ri';
+
+import './InfoPanel.css';
+
+const InfoPanel = () => {
+  const uploadExcelHandler = (e) => {
+    e.preventDefault();
+    console.log(`File was sent!`);
+  };
+
+  return (
+    <section className="info-panel">
+      <h1 className="info-panel__main-heading">ExcQL</h1>
+      <p className="info-panel__description">
+        Lorem ipsum, dolor sit amet consectetur adipi sicing elit. Volupta tibus
+        aliquid pariatur enim vitae accusantium qua erat cum! Nisi totam
+        deserunt labore, aliquam repellat repudiandae perspiciatis explicabo
+        pariatur ducimus commo di nihil velit! Lorem ipsum dolor sit amet
+        consectetur adipisicing elit. Dolor natus volup tate beatae fugit quidem
+        accusamus provident nostrum sequi quas!
+      </p>
+      <form
+        action="/test"
```

Comment:
Does this API endpoint work currently? I assume not? If so we may want to open a task in Asana to connect the front and back end to make sure we do not miss this.

Reply:
Hey Nattie, No. This endpoint is a dummy endpoint. In order for it to work so the backend team can use it to send themselves the file through the front end, I'd need to add a few lines of code. I can do this tonight.